src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -164,6 +167,20 @@ class PaymentMethodList extends Component {
         return combinedPaymentMethods;
     }
 
+    /**
+     * Dismisses the error on the payment method
+     * @param {*} item
+     */
+    dismissError(item) {
+        const paymentList = item.accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? ONYXKEYS.BANK_ACCOUNT_LIST : ONYXKEYS.CARD_LIST;
+        const paymentID = item.accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? lodashGet(item, ['accountData', 'bankAccountID']) : lodashGet(item, ['accountData', 'cardID']);
```

Comment:
I see you are using `lodashGet` to get `['accountData', 'bankAccountID']`, so that makes me think that you are considering these things could be missing... but if they are missing you don't have a default, so you get `undefined`, right? what happens then if you call `PaymentMethods.clearDeletePaymentMethodError` or `PaymentMethods.clearAddPaymentMethodError` with an `undefined` paymentID? I don't think it crashes, but you would a weird key `undefined` in Onyx.

My advice would be:
- remove `lodashGet` and let it crash if the info was not there, or
- add a check `if(paymentID){return;}` to exit if it didn't find it (and log?)

Reply:
I decided to just add a check if paymentID doesn't exist - We shouldn't let crashes happen but i imagine if the user presses this quickly twice the paymentID would be empty

Reply:
Sounds good to me!